### PR TITLE
Immunize, listings

### PR DESCRIPTION
**Pull request by adriandumavitagroup**

### CODE REVIEW

File: src/app/immunizations/immunization-components/immunization-profile/immunization-profile-form-data.ts [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+      },
+      {
+        ...FORM_DATA_DATE,
+        key: 'endDate',
+        label: 'endDate',
+      },
+      {
+        ...FORM_DATA_DATE,
+        key: 'validFrom',
+        label: 'Valid from',
```
Comment (danielrus) [inline src/app/immunizations/immunization-components/immunization-profile/immunization-profile-form-data.ts]: translate

File: src/app/immunizations/immunization-components/immunization-profile/immunization-profile-form-data.ts [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+        newLine: true,
+      },
+      {
+        ...FORM_DATA_DATE,
+        key: 'validUntil',
+        label: 'Valid until',
```
Comment (danielrus) [inline src/app/immunizations/immunization-components/immunization-profile/immunization-profile-form-data.ts]: translate

File: src/app/immunizations/immunization-components/immunization-profile/immunization-profile-form-data.ts [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+        ...FORM_DATA_SELECT,
+        key: 'disease',
+        placeholder: 'captions.disease',
+        options: diseaseOptions,
+        validation: ['required'],
+      },
+      {
+        ...FORM_DATA_RADIO,
+        key: 'previousInfection',
+        label: 'Previous infection with this disease',
```
Comment (danielrus) [inline src/app/immunizations/immunization-components/immunization-profile/immunization-profile-form-data.ts]: translate

File: src/app/immunizations/immunization-components/immunization-profile/immunization-profile-form-data.ts [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+    ],
+  },
+  {
+    id: 'country',
+    title: 'captions.country',
+    fields: [
+      {
+        ...FORM_DATA_SELECT,
+        key: 'country.uuid',
+        label: 'immunization country',
```
Comment (danielrus) [inline src/app/immunizations/immunization-components/immunization-profile/immunization-profile-form-data.ts]: translate

File: src/app/immunizations/immunization-filters/immunization-filters-form-data.ts [hunk 5]
```diff
@@ -144,15 +209,22 @@ export const FORM_DATA_IMMUNIZATION_FILTERS = [
...
       {
         ...FORM_DATA_DATE,
         key: 'newImmunizationDateTo',
-        hint: 'strings.date.to',
+        hint: 'From',
+        className: 'fullwidth',
+      },
+      {
+        ...FORM_DATA_DATE,
+        key: 'newImmunizationDateTo222',
```
Comment (danielrus) [inline src/app/immunizations/immunization-filters/immunization-filters-form-data.ts]: is this key correct?

**Review by sandorbanvitagroup — APPROVED** [review]

**Review by danielrus — APPROVED** [review]

**Pull request merged**